ctoec/data-collection
# Redesign initial upload page

## Background
This PR redesigns the initial upload page.

## GitHub Issue
https://github.com/ctoec/data-collection/issues/1380

## Associated PRs
https://github.com/ctoec/component-library/pull/170

## Validation Plan
* head to the upload page
* ensure designs match design specs
* ensure file upload still works

## Automated Testing
- [ ] If applicable, unit tests have been added to cover this changeset.

## Additional Context
N/A
